src/main/java/com/ioet/bpm/skills/boundaries/SkillController.java

```diff
@@ -53,16 +57,11 @@ public SkillController(SkillRepository skillRepository) {
     @PutMapping("/{id}")
     public ResponseEntity<Skill> updateSkill(@PathVariable(value = "id") String skillId,
                                              @Valid @RequestBody Skill skillDetails) {
-
-        Optional<Skill> skillOptional = skillRepository.findById(skillId);
-        if (!skillOptional.isPresent()) {
+        try{
+            return new ResponseEntity<>(skillService.save(skillId,skillDetails),  HttpStatus.OK);
+        }
+        catch (EntityNotFoundException ex){
```

Comment:
Please get rid of this exception, check the comments I added in @juanultimate's PR

Reply:
Since this is part of the code @juanultimate wrote, I'll wait until his changes are in master and then integrate those into my code.

Reply:
Done